<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClipboardManager</class>
 <widget class="QMainWindow" name="ClipboardManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clipboard Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/clipboard-manager.ico</normaloff>icons/clipboard-manager.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 219, 208);
border-radius: 10px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(68, 44, 46);</string>
    </property>
    <property name="text">
     <string>Clipboard</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 234, 230);
color: rgb(68, 44, 46);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionEmail_Extractor"/>
    <addaction name="actionURL_Extractor"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
    <addaction name="actionCheck_History"/>
    <addaction name="actionClear_History"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHistory"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEmail_Extractor">
   <property name="icon">
    <iconset>
     <normaloff>icons/webmail.png</normaloff>icons/webmail.png</iconset>
   </property>
   <property name="text">
    <string>Email Extractor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionURL_Extractor">
   <property name="icon">
    <iconset>
     <normaloff>icons/url.png</normaloff>icons/url.png</iconset>
   </property>
   <property name="text">
    <string>URL Extractor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>icons/exit.ico</normaloff>icons/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q, Alt+F4</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>icons/about.png</normaloff>icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionCheck_History">
   <property name="icon">
    <iconset>
     <normaloff>icons/history.png</normaloff>icons/history.png</iconset>
   </property>
   <property name="text">
    <string>Check History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionClear_History">
   <property name="icon">
    <iconset>
     <normaloff>icons/clear-history.png</normaloff>icons/clear-history.png</iconset>
   </property>
   <property name="text">
    <string>Clear History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>icons/new.png</normaloff>icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
